app/assets/stylesheets/application.scss

```
@@ -14,5 +14,5 @@
 // Layouts
 
 // Components
-
+@import './components/search_stat_details'
```

Add an empty line after each section.

```suggestion
@import './components/search_stat_details'

```